### PR TITLE
Feat: output the connection string after the reset

### PR DESCRIPTION
**Pull request by antelman107**

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -18,11 +18,32 @@ inputs:
   parent:
     description: 'If specified, the branch will be reset to the parent branch'
     required: false
+  cs_username:
```
Comment (duskpoet) [inline action.yml]: why not more traditional "role name"?

Comment (antelman107) [reply action.yml]: renamed cs_username -> cs_role_name.
If you asking why there is cs_ prefix - it's to understand that all those parameters are needed for cs subcommand or for connection string. 
In create-branch-action those parameters are without prefix and I was misled couple of times, thinking that they are used to create the branch, but no - same as here - they are needed only to get the connection string.

File: README.md [hunk 2]
```diff
@@ -37,8 +42,27 @@ outputs:
   branch_id:
     description: 'Reset branch id'
     value: ${{ steps.reset-branch.outputs.branch_id }}
+  db_url:
+    description: 'DATABASE_URL of the branch after the reset'
+    value: ${{ steps.reset-branch.outputs.db_url }}
+  db_url_with_pooler:
+    description: 'DATABASE_URL with pooler of the branch after the reset'
+    value: ${{ steps.reset-branch.outputs.db_url_with_pooler }}
+  cs_role_name:
```
Comment (duskpoet) [inline README.md]: I don't see where these values are added as outputs. I am not sure if it makes sense, too.

Comment (antelman107) [reply README.md]: I've added `host`, `host_with_pooler`, `password` outputs - same as for create-branch-action.

File: test.sh [hunk 3]
```diff
@@ -0,0 +1,10 @@
+echo "dsdsd-$(echo 5)"
```
Comment (duskpoet) [inline test.sh]: if you intend to leave this file, please write instruction on how to use it in comments

**Review by duskpoet — APPROVED** [review]

**Pull request merged**